arc3.cfg

```diff
@@ -111,19 +111,15 @@ query_name = eclipse
   scs128 = SCS 128       : SCS 128   : \-Y SA
   scs129 = SCS 129       : SCS 129   : \+Y SA
   scs130 = SCS 130       : SCS 130   : \-Y SA
-  scs131 = SCS 131       : SCS 131   : E1300
-  scs132 = SCS 132       : SCS 132   : EPH A-Leak
-  scs133 = SCS 133       : SCS 133   : EPH B-Leak
+  scs131 = SCS 131       : SCS 131   :  Pitch Mom.
+  scs132 = SCS 132       : SCS 132   : Ephin
+  scs133 = SCS 133       : SCS 133   : EIO
   scs107 = SCS 107       : SCS 107   : EPH temp
   hrcshield = HRC shield : SHLD RT   : FSS Alfa
   hrcmcp = HRC MCP       : EVT RT    : AOACSTAT
-  E150   = E150          : E150      : SCS 130
-  E1300  = E1300         : E1300     : Pitch Mom
-  P4GM   = P4GM          : P4GM      : Yaw Mom
-  P41GM  = P41GM         : P41GM     : EPH temp
```

Comment:
@taldcroft I've forgotten the history here; should values for these be fetched from a non-Snapshot if they are no longer in the snapshot?

Reply:
I guess the P4GM and P41GM values are grabbed from GOES, but I'm not sure what the CXO values for E150 and E1300 should be at this point around

https://github.com/sot/arc/blob/9b314477240e03428b2fab6a5d342233f8391a57/arc.pl#L824

Reply:
EPHIN is no more, off, kaput.  😢 :  So the E150 and E1300 table rows can be deleted entirely. :smile:

Reply:
I think I'll leave the CXO values as '---' for now so until I understand the formatting issues later.

Reply:
Yes, `---` is appropriate since there are no EPHIN (CXO) values but we do have meaningful proxies.

Reply:
I meant for the P4 ones.

Reply:
And yes, I figured.  And I'd originally meant to just use '---' for the E150 and E1300 rows, but I figured out where they live in the page template so have cut them successfully ineb1d586612afe1057d4212201d5e90fd806885b4